Documentation-Requirements.md

```diff
@@ -20,6 +21,18 @@ All documents must:
 - Include a TOC (table of contents) at the top of the document with links to
   all heading sections.
 
+# Linking advice
+
+Linking between documents is strongly encouraged to help users and developers
+understand the material more easily. Linking also avoids repetition - if a
```

Comment:
s/understand/navigate/ ? just a thought

Reply:
I've updated it with your suggestion as that probably makes more sense to most folk. But in my mind it's both: the linking between docs makes it easier to comprehend what this project is all about ;)